### PR TITLE
Two Innovation Hubs e2e

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: alkemio.yml [hunk 1]
```diff
@@ -74,6 +74,9 @@ security:
     # Cors allowed headers.
     allowed_headers: ${CORS_ALLOWED_HEADERS}:Origin,X-Requested-With,Content-Type,Accept,Authorization
 
+innovation_hub:
+  header: ${INNOVATION_HUB_HEADER}:myhost
```
Comment (valentinyanakiev) [inline alkemio.yml]: change the default to the real header used

File: src/domain/challenge/hub/hub.resolver.queries.ts [hunk 2]
```diff
@@ -15,8 +19,15 @@ export class HubResolverQueries {
...
-    return await this.hubService.getHubs(args);
+  async hubs(
+    @InnovationHubDecorator() innovationHub: InnovationHub | undefined,
+    @Args({ nullable: true }) args: HubsQueryArgs
+  ): Promise<IHub[]> {
+    if (!innovationHub) {
+      return await this.hubService.getHubs(args);
+    }
+
+    return this.hubService.getHubsForInnovationHub(innovationHub);
```
Comment (valentinyanakiev) [inline src/domain/challenge/hub/hub.resolver.queries.ts]: maybe a more appropriate name would be `getSpacesForInnovationHub`. It is getting increasingly confusing.

File: src/domain/challenge/hub/hub.service.ts [hunk 3]
```diff
@@ -286,6 +288,61 @@ export class HubService {
     return hub.visibility;
   }
 
+  public getHubsForInnovationHub({
```
Comment (valentinyanakiev) [inline src/domain/challenge/hub/hub.service.ts]:
```suggestion
  public getSpacesForInnovationHub({
```

Comment (valentinyanakiev) [reply src/domain/challenge/hub/hub.service.ts]: Don't change the rest of the methods yet but this one at least would make sense

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Comments inline

File: src/domain/challenge/hub/hub.resolver.queries.ts (hunk 2 above)
Comment (hero101) [reply src/domain/challenge/hub/hub.resolver.queries.ts]: I was hesitant to start using the new convention, but if you also believe so, I will start.

**Pull request merged**